apps/docs/src/content/docs/_meta.ts

```diff
@@ -19,11 +19,15 @@ export default {
     type: "separator",
     title: "Foundation",
   },
-  theming: "Theming",
+  "ui-kit-tokens": {
+    title: "UI Kit Tokens",
```

Comment:
can we call this "theme tokens"? at least in the URL, it looks weird/redundant

```suggestion
  "theme-tokens": {
    title: "Theme Tokens",
```